nrich-security-csrf/README.md

```diff
@@ -53,7 +56,7 @@ WebFlux environments.
 
         };
     }
-
+}
 
 ```
```

Comment:
At line 67, maybe replace `is a controller that will be used as a ping call for Csrf token.` with `is a controller that will be used for a ping call for Csrf token.`

At line 69, `it is an implementation...` should be a new sentence. I would change second part into something like `which intercepts every request and verifies token for every request except excluded
list`. Part with `and adding generated token for initial token url` should be extracted into a new sentence.

At line 73, ` it is an implementation...` should be a new sentence and second part `it has...` should be changed to `and has...`.